includes/translations.php

```diff
@@ -1,9 +1,10 @@
 <?php
 
-header('Content-type: application/x-javascript; encoding=utf8');
+header( 'Content-type: application/x-javascript; encoding=utf8' );
 
 $lang = $_GET['lang'];
-if (!$lang) $lang = 'en';
+if ( !$lang ) { $lang = 'en';
+}
```

Comment:
The if statement look weird.
Either it's a one-liner, in which case the } should be on the same line, or it's a block. Here it's a mixture.